src/apis/token-swap.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios'
+
+const baseApiUrl = `${process.env.REACT_APP_TOKEN_SWAP_API_URL}`
+
+/**
+ * Get quotes from liquidity providers
+ * @typedef {Object} data
```

Comment:
https://jsdoc.app/tags-param.html#parameters-with-properties We tend to comment objects like this and use capital letters for types

Reply:
So I understand that we prefer params to define objects instead typedef and property?